code/modules/events/zombie_outbreak.dm

```diff
@@ -0,0 +1,31 @@
+/datum/round_event_control/zombie_outbreak
+	name = "Zombie Outbreak"
+	typepath = /datum/round_event/zombie_outbreak
+	weight = 5
+	max_occurrences = 1
+	min_players = 30
+
+/datum/round_event/zombie_outbreak
+	fakeable = FALSE
+
+/datum/round_event/zombie_outbreak/start()
+	// The amount of infections among the crew depends on the amount of living, non-AFK, non-silicon players on the station
+	var/list/living_crew = list()
+	living_crew = get_living_station_crew()
+	var/infection_count = round(length(living_crew) / 10)
+
+	for(var/i = 0, i < infection_count, i++)
```

Comment:
```suggestion
	for(var/i in 0 to infection_count)
```
Did you know this is actually slightly faster? The more you know

Reply:
done